api/v1.inc

```diff
@@ -45,6 +45,7 @@ $router->add_route("GET", "v1/projects/:projectid/pagedetails", "api_v1_project_
 $router->add_route("GET", "v1/projects/:projectid/pages", "api_v1_project_pages");
 $router->add_route("PUT", "v1/projects/:projectid/pages/:pagename", "api_v1_project_page");
 $router->add_route("GET", "v1/projects/:projectid/pages/:pagename", "api_v1_project_page");
+$router->add_route("PUT", "v1/projects/:projectid/pages/:pagename/report_bad", "api_v1_project_page_report_bad");
```

Comment:
We haven't used `_`s in any of the endpoints so far, despite things like "imagesources" and "specialdays" so let's not start now:

```suggestion
$router->add_route("PUT", "v1/projects/:projectid/pages/:pagename/reportbad", "api_v1_project_page_report_bad");
```

(with corresponding update in `dp-openapi.yaml`)